04.ls/ls.rb

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+ORDER_COLUMN = 3
+BLANK = 2
+
+def main
+  files = Dir.glob('*')
+
+  remainder = files.size % ORDER_COLUMN
+  padding_count = (ORDER_COLUMN - remainder) % ORDER_COLUMN
+  paddings = [''] * padding_count
+  padded_files = files + paddings
+  rows_count = padded_files.size / ORDER_COLUMN
+  files_grid = padded_files.each_slice(rows_count).to_a
+  display_files = files_grid.transpose
```

Comment:
```suggestion
  file_grid = padded_files.each_slice(rows_count).to_a.transpose
```

あまり変数を小分けにしすぎると、変数名がネタ切れしそうになるので、可読性を損なわない範囲でメソッドチェーンを使うのもいいかもです。

---

`display_files`は「ファイルを表示する」というメソッドっぽく見えるのでちょっといまいちかも。
https://qiita.com/jnchito/items/459d58ba652bf4763820

そもそもこの変数は「配列の配列（ネストした配列）」なので、単なる複数形よりも、gridとかtableみたいに呼んであげた方が、データの構造がイメージしやすいですね。（ただの複数形だと、ただの配列に見える）

---

row_countの件と同じで、修飾語として使う名詞は単数形でいいと思います。
files_grid → file_grid